File: Java/Ejercicios01/Empleado.java
public class Empleado extends Persona{
    private double salario;
    
    public Empleado(String nombre, Fecha fecNac, double salario){
        super(nombre, fecNac);
        this.salario = salario;
    }

    public Empleado(){}

    public void imprimeDescripcion(){
        System.out.println("Soy una clase Empleado");
        System.out.println("Con salario: " + salario);
    }

    public void setSalario(double salario) {
        this.salario = salario;
    }

    public double getSalario() {
        return salario;
    }

    public double calcQuincena(){
        return salario * 15;
    }
}

File: Python/Ejercicios02/Empleado.py
from Fecha import *
class Empleado:
    def __init__(self, nombre = str, salario = float, fecNac = Fecha):
        self.__nombre = nombre
        self.__salario = salario
        self.__fecNac = fecNac

    def setNombre(self, nombre):
        self.__nombre = nombre

    def getNombre(self):
        return self.__nombre
    
    def setSalario(self, salario):
        self.__salario = salario

    def getSalario(self):
        return self.__salario
    
    def setFecNac(self, fecNac):
        self.__fecNac = fecNac

    def getFecNac(self):
        return self.__fecNac
    
    def calcQuincena(self):
        return self.__salario * 15